frontend/src/redux/store.js

```
@@ -2,9 +2,25 @@ import { configureStore } from '@reduxjs/toolkit';
 import userSlice from './userSlice';
 import { appMessageReducer } from './globalComponentSlice';
 
+import { configureStore } from '@reduxjs/toolkit';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'configureStore'. Consider to delete it or rename it.

'configureStore' is defined here:


(lint/suspicious/noRedeclare)